test/Interop/ObjCToSwiftToObjCxx/bridge-objc-types-back-to-objcxx-execution.mm

```diff
@@ -24,6 +24,16 @@ -(ObjCKlass * _Nonnull) init:(int)x;
 -(int)getValue;
 @end
 
+@protocol ObjCProtocol
```

Comment:
This test does not run by the CI, unfortunately. It is disabled. But I enabled it locally and it does pass for me. I do not want to reenable this as part of this PR because I don't think the original failure was ever fixed (on a platform different from what I am running on).